src/app/stacks/multistack_robocup_ssl.cpp

```diff
@@ -84,14 +86,16 @@ MultiStackRoboCupSSL::MultiStackRoboCupSSL(RenderOptions * _opts, int cameras) :
       *global_field);
 
   //add parameter for number of cameras
-  createThreads(cameras);
-  unsigned int n = threads.size();
-  for (unsigned int i = 0; i < n;i++) {
+  int num_threads = num_normal_camera_threads + 1;
+  createThreads(num_threads);
+  std::vector<CaptureSplitter*> captureSplitters;
+  captureSplitters.resize((unsigned long) num_normal_camera_threads);
+  for (unsigned int i = 0; i < num_normal_camera_threads;i++) {
     //NOTE: if modified to put different stacks in cameras, please
     //      update the plugin_colorcalib.cpp code to safely reallocate and copy their
     //      data instead of assuming that format and size is uniform across
     //      cameras -- added when LUTs became aware of other cameras (Zavesky, 2/16)
-    threads[i]->setFrameBuffer(new FrameBuffer(5));
+    threads[i]->setFrameBuffer(new FrameBuffer(3));
```

Comment:
Why 3?

Reply:
That's the minimum. With less, the application will block.